weaver/wps.py

```diff
@@ -185,4 +185,7 @@ def includeme(config):
         wps_path = get_wps_path(settings)
         config.add_route("wps", wps_path)
         config.add_view(pywps_view, route_name="wps")
+
+        config.add_static_view(get_wps_output_path(config), get_wps_output_dir(config))
```

Comment:
this shouldn't be required
will cause problems with nginx that is currently doing it

Reply:
Wouldn' t nginx override this? The way I see it it doesn't cause any harm because nginx catches the route before weaver does. And adding this makes running a local weaver without the need to have nginx running.

Reply:
Good point David.

Reply:
Also, isn't this the idea for #5?

Reply:
If it works in both cases, great.
Actually, we could remove it altogether from nginx, because `weaver.wps_output_dir` and `weaver.wps_output_path` are hardcode in it.
This variant is better if workflows are functional with it.

Reply:
nginx would serve the static content 'faster' but I don't know by how much.

Reply:
Still worth it to move toward #5. So let's go with your solution and directly remove the config in nginx while we are at it.

Reply:
I do not agree to remove the file serving by nginx. We are talking of huge netcdf file and we should avoid overhead here...

Reply:
But it could be a problem, since one could set `weaver.wps_output_path = "/my-new-path` and wonder for a long time why behavior is not as expected.

Reply:
I didn't think about large files... wouldn't nginx be more appropriate for large files? 

Also, wouldn't setting `weaver.wps_output_path = "/my-new-path"` break the correct saving of outputs anyway?

I think we can have both... keep the static route in weaver, remove nginx in the config, but when we deploy, configure another instance of nginx to serve static content on a specific route.

Reply:
The idea with `weaver.wps_output_path` is to specify which weaver url to call to obtain the process results. I couldn't find a way to cleanly match nginx's, pywps's and weaver's configs together under this same setting. So currently, it just "works" because the values are all the same...
I managed to somehow patch the pywps/weaver configs with this setting, but not nginx. I was counting on #5 to just drop it.

Reply:
There's not only large file, there is simultaneously access... Nginx will serve multiples files at the same time without issues while we will rapidly run out of python workers all strugling to serve the files.
For the config issue why not using an environment variable both used in the nginx and weaver config files with the template setup we have?

Reply:
An external nginx in the compose to serve the files is an interesting idea.
We would need to make sure that `add_static_view` works standalone so that anyone can still run weaver by itself.

Reply:
The external nginx is already serving the files using the volume wps_outputs. 
so I think we got a consensus that #5 is the way to go and adding the static serve is helpful in debugging and not harmful in production. Config wise using the compose volume wps_outputs and mounting it in both docker image at a proper place should do it.